include/exadg/incompressible_navier_stokes/user_interface/enum_types.h

```diff
@@ -612,6 +626,28 @@ std::string
 enum_to_string(SchurComplementPreconditioner const enum_type);
 
 
+
+/**************************************************************************************/
+/*                                                                                    */
+/*                            SOLVE MASS SYSTEM (projection)                          */
+/*                                                                                    */
+/**************************************************************************************/
+
+/*
+ *  Preconditioner type for solution of mass system, i.e a projection:
```

Comment:
I think we should avoid "projection" here (and above), since it is used in a different context in case of the Navier-Stokes solvers. There should somewhere also be a comment explaining that these parameters and the available preconditioners are relevant only for the HDIV case.